v4.4.0 release notes (#1422)
mesosphere/kubernetes-base-addons: RELEASE_NOTES.md

# Release Notes

## stable-1.20-4.4.0, v4.4.0

### kube-oidc-proxy

>   - * bump kube-oidc-proxy to 0.3.0

Reviewer: ```suggestion
  - Bumps kube-oidc-proxy to 0.3.0 to resolve "kubectl log" latency issues (COPS-7123)
```

Author: Please update the PR release notes and I'll run the generator again.

Reviewer: ✅

  #1420 (@mesosphere-mergebot)

### nvidia

>   - Fixes Nvidia GPU platform service placement constraints.

Reviewer: ```suggestion
  - Fixes Nvidia GPU platform service placement constraints (COPS-7142)
```

  #1417 (@mesosphere-mergebot)

>   - updates Nvidia DCGM exporter to 2.2.9

Reviewer: ```suggestion
  - Updates Nvidia DCGM exporter to 2.2.9 to fix metrics (COPS-7132)
```

  #1412 (@mesosphere-mergebot)

### prometheus

>   - [prometheus] Prometheus addon upgrades now properly upgrade the alertmanager CRD (COPS-6842)
>   #1330 (@mesosphere-mergebot)

Reviewer: ```suggestion
  - Alertmanager CRD is now properly upgraded with Prometheus addon (COPS-6842)
  #1330 (@mesosphere-mergebot)

  - Adds concurrency policy Replace configuration to the home dashboard CronJob to avoid potentially creating an unbound number of pods (COPS-7105)
  #1399 (@mesosphere-mergebot)
```
